modules/library/main/src/main/java/org/geotools/styling/ColorMapEntry.java

```diff
@@ -34,10 +34,10 @@
  */
 public interface ColorMapEntry {
     /** Label for this Color Map Entry */
-    String getLabel();
+    Expression getLabel();
```

Comment:
Big nope, this is a core API change, it should have its own proposal and would make this PR non backportable. If you need expressions instead of literals, it can be done, but make it a separate commit, after issuing a proposal and getting it voted.

Reply:
brought back to string